src/components/JTextArea/JTextArea.js

```diff
@@ -28,37 +44,91 @@ function heightCalc({ currentTarget: target }: SyntheticEvent<HTMLTextAreaElemen
   }
 }
 
-export class JTextArea extends PureComponent<Props, void> {
+export class JTextArea extends PureComponent<Props, State> {
   static defaultProps = {
-    onBlur: undefined,
-    onFocus: undefined,
-    theme: 'blue',
+    onChange: noop,
+    onBlur: noop,
+    onFocus: noop,
+    theme: 'white',
     className: undefined,
+    label: undefined,
+    id: undefined,
+    value: undefined,
+    rows: 1,
+    error: false,
+    disabled: false,
+  }
+
+  constructor(props: Props) {
+    super(props)
+
+    this.state = {
+      focused: String(this.props.value).length > 0,
+    }
   }
 
-  handleChange = (fieldEvent: SyntheticEvent<HTMLTextAreaElement>): void => {
-    heightCalc(fieldEvent)
-    this.props.onChange(fieldEvent)
+  handleChange = (event: SyntheticEvent<HTMLTextAreaElement>): void => {
+    heightCalc(event)
+    this.props.onChange(event)
+  }
+
+  handleBlur = (event: SyntheticEvent<HTMLTextAreaElement>): void => {
+    if (String(this.props.value).length <= 0) {
+      this.setState({ focused: false })
+    }
+
+    this.props.onBlur(event)
+  }
+
+  handleFocus = (event: SyntheticEvent<HTMLTextAreaElement>): void => {
+    this.setState({ focused: true })
+    this.props.onFocus(event)
   }
 
   render() {
     const omitedProps = omit(this.props, [
```

Comment:
omited -> omitted

but would be better to name it for example restProps/otherProps/remainingProps